src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1348,11 +1348,13 @@ private function registerAssetMapperConfiguration(array $config, ContainerBuilde
         foreach ($config['excluded_patterns'] as $path) {
             $excludedPathPatterns[] = Glob::toRegex($path, true, false);
         }
+        $strictMode = $container->getParameter('kernel.debug');
 
         $container->getDefinition('asset_mapper.repository')
             ->setArgument(0, $paths)
             ->setArgument(2, $excludedPathPatterns)
-            ->setArgument(3, $config['exclude_dotfiles']);
+            ->setArgument(3, $config['exclude_dotfiles'])
+            ->setArgument(4, $strictMode);
```

Comment:
can be reverted, the change on the config is enough

Reply:
Done!